src/plugins/intel_cpu/src/nodes/rope.cpp

```diff
@@ -45,7 +45,7 @@ struct RoPE::RoPEExecutorRotateHalf : public RoPE::Executor {
             t_src = t_src.slice(3, config.slice_start, config.slice_stop);
         }
         if (config.input_trans0213) {
-            t_src = t_src.permute({0, 2, 1, 3});
+            t_src = t_src.permute({0, 2, 1, 3*1});
```

Comment:
this harmless change is for triggering CI's CPU functional test on windows to ensure that the change does work indeed.